frontend/src/pages/forms/Forms.jsx

```diff
@@ -160,12 +172,128 @@ const Forms = () => {
+      return {
+        [id]: findCascade ? [findCascade.id] : [],
+      };
+    },
+    [authUser?.administration?.level]
+  );
+
+  const transformValue = (type, value) => {
+    if (type === "option" && Array.isArray(value) && value.length) {
+      return value[0];
+    }
+    if (type === "geo" && Array.isArray(value) && value.length === 2) {
+      const [lat, lng] = value;
+      return { lat, lng };
+    }
+    return typeof value === "undefined" ? "" : value;
+  };
+
+  const fetchInitialMonitoringData = useCallback(
+    async (response) => {
+      try {
+        const { data: apiData } = response;
+        const questions = apiData?.question_group?.flatMap(
+          (qg) => qg?.question
+        );
+        const res = await fetch(
+          `${window.location.origin}/datapoints/${monitoring?.uuid}.json`
+        );
+        const { answers } = await res.json();
+        /**
+         * Transform cascade answers
+         */
+        const cascadeAPIs = questions
+          ?.filter((q) => q?.type === "cascade" && q?.api?.endpoint)
+          ?.map((q) => getCascadeAnswerId(q.id, q.api, answers?.[q.id]));
+        const cascadeResponses = await Promise.allSettled(cascadeAPIs);
+        const cascadeValues = cascadeResponses
+          .filter(({ status }) => status === "fulfilled")
+          .map(({ value }) => value)
+          .reduce((prev, curr) => {
+            const [key, value] = Object.entries(curr)[0];
+            prev[key] = value;
+            return prev;
+          }, {});
+        /**
+         * Transform answers to Webform format
+         */
+        const initialValue = questions.map((q) => {
+          let value = Object.keys(cascadeValues).includes(`${q?.id}`)
+            ? cascadeValues[q.id]
+            : transformValue(q?.type, answers?.[q.id]);
+          // set default answer by default_value for new_or_monitoring question
+          if (
+            q?.name === "new_or_monitoring" &&
+            q?.default_value?.submission_type?.monitoring
+          ) {
+            value = q.default_value.submission_type.monitoring;
+          }
+          return {
+            question: q?.id,
+            value: value,
+          };
+        });
+        store.update((s) => {
+          s.initialValue = initialValue;
+        });
+      } catch (error) {
+        Modal.error({
+          title: text.updateDataError,
+          content: String(error),
+        });
+      }
+    },
+    [getCascadeAnswerId, monitoring?.uuid, text.updateDataError]
+  );
+
   useEffect(() => {
-    if (formId && loading) {
+    if (isEmpty(forms) && formId) {
       api.get(`/form/web/${formId}`).then((res) => {
         const questionGroups = res.data.question_group.map((qg) => {
           const questions = qg.question.map((q) => {
             let qVal = { ...q };
+            // set initial value for new_or_monitoring question
+            if (
+              q?.name === "new_or_monitoring" &&
```

Comment:
if question has default_value, Instead of new_or_monitoring

Reply:
Updated mas.